sklearn/pipeline.py

```diff
@@ -531,6 +531,20 @@ def _pairwise(self):
         # check if first estimator expects pairwise input
         return getattr(self.steps[0][1], '_pairwise', False)
 
+    def get_feature_names(self, input_features=None):
```

Comment:
this is the actual implementation that enables everything. It's pretty short and would be even shorter if we force ``get_feature_names`` to accept ``input_features`` (which I think we should).